src/views/pages/ManagePeople/index.vue

```diff
@@ -52,81 +53,73 @@ export default Vue.extend({
       this.searched = searchByName(this.people, this.search);
     },
   },
-  created: function created() {
-    this.searched = this.people;
+  created: async function created() {
+    const self = this;
+
+    // get people types
+    const personCategories: any[] = [];
+    await axios.get('http://boresha.tech:9090/people/categories')
+      .then(async function (response: any) {
+        response.data.forEach(function(category: any) {
+          personCategories.push(category.name);
+        });
+      })
+      .catch(async function (error: any) {
+        console.log(error.message);
+      });
+
```

Comment:
Have you investigated how forEach works with async functions? Any code you write after the forEach call won't necessarily occur after code written in your async function. To add more clarity I would advise using `map` instead, wrap in Promise.all and await that unless this behaviour was intentional